onnxruntime/core/providers/cuda/tensor/tile_impl.cu

```diff
@@ -66,7 +66,32 @@ void TileImpl(cudaStream_t stream, const size_t shape_rank, const TArray<fast_di
 }
 
 template <typename T>
-__global__ void _TileMemcpyKernel(const T* input_data, T* output_data, const CUDA_LONG N, const size_t repeats) {
+__global__ void _TileMemcpyKernelFromOutput(const T* input_data, T* output_data,
+                                            const fast_divmod divmod_num_input_elements, const CUDA_LONG N) {
+  CUDA_LONG start = num_elements_per_thread * num_threads_per_block * blockIdx.x + threadIdx.x;
```

Comment:
>UDA_LONG start = num_elements_per_thread * num_threads_per_block * blockIdx.x + threadIdx.x; [](http://example.com/codeflow?start=3&length=92)

can we use "CALCULATE_ELEMENTWISE_INDEX_OR_EXIT(id, N);", or we do a < N check ,before doing the subsequent  loops

Reply:
N = num_threads * 4, so CALCULATE_ELEMENTWISE_INDEX_OR_EXIT cannot be used here.

Reply:
Yeah, I missed it.